README.md

```diff
@@ -15,7 +15,7 @@
 Ruby 2.3.1, Docker, MySQL, SQLite 3, Redis, mitmproxy.
 (See below for more details about installing Docker)
 
-Development is supported on Linux and Mac OS X.
+Development is supported on Linux (Ubuntu 16.04 works best; Ubuntu 18.04 is possible with some setup) and Mac OS X.
```

Comment:
I think the changes I've made take care of this, so this comment can probably be reverted.

Reply:
Then revert it ;-)

Reply:
Checking again - still required, so I'll leave it.